README.md

```diff
@@ -212,7 +212,7 @@ Run the assembly.
 ```sh
 mkdir k48
 ln -s ../30CJCAAXX_4_1.fq.gz ../30CJCAAXX_4_2.fq.gz k48/
-abyss-pe -C k48 name=HS0674 k=48 v=-v in="30CJCAAXX_4_1.fq.gz 30CJCAAXX_4_2.fq.gz" contigs 2>&1 | tee abyss.log
+abyss-pe -C k48 name=HS0674 s=200 k=48 v=-v in="30CJCAAXX_4_1.fq.gz 30CJCAAXX_4_2.fq.gz" contigs 2>&1 | tee abyss.log
```

Comment:
Please move *s* after *k* in each occurrence. When choosing an arbitrary order, alphabetical order is a good choice when no logical order presents itself.

Reply:
```suggestion
abyss-pe -C k48 name=HS0674 k=48 s=200 v=-v in="30CJCAAXX_4_1.fq.gz 30CJCAAXX_4_2.fq.gz" contigs 2>&1 | tee abyss.log
```

Reply:
Done! I will add a new pull request.

Reply:
Sorry, I mean I did it in this active pull request. Done!